### ISSUE TITLE
Semantic Tag functionality is missing

### ISSUE THREAD

**Issue by uskudarli** [awesome-possums/swe574#105]
I can tag heritage items with regular textual labels. But not semantic tags.

**Comment by irmakcakmak** [comment]
@uskudarli That is true. It is also not possible to observe any requirement or design item related to this feature. We have completely missed the mark on this one.

What should we do? What is your expectation and deadline for it?

**Comment by uskudarli** [comment]
Yes, I think it got lost somehow. We did discuss it several times, alas, got lost in the technical documents sadly.

I am just documenting it. It does not seem feasible to do anything at this point.

Thank you for your response.
